emctl/cmd/client/command/printer/printer.go

```diff
@@ -18,32 +18,46 @@
 package printer
 
 import (
-	"encoding/json"
 	"fmt"
 	"os"
+	"strings"
 
-	yamljsontool "github.com/ghodss/yaml"
-	"github.com/megaease/easemeshctl/cmd/client/resource"
+	"github.com/megaease/easemeshctl/cmd/client/resource/meta"
 	"github.com/megaease/easemeshctl/cmd/common"
 	"github.com/olekukonko/tablewriter"
+
+	jsoniter "github.com/json-iterator/go"
+	"gopkg.in/yaml.v2"
 )
 
 type (
 	// Printer prints information about the EaseMesh objects
 	Printer interface {
-		PrintObjects(objects []resource.MeshObject)
+		PrintObjects(objects []meta.MeshObject)
 	}
+
 	printer struct {
 		outputFormat string
 	}
+
+	TableColumn struct {
```

Comment:
Need be changes according to complain of the GitHub Actions

Reply:
Fixed all